acoustid/server.py

```
@@ -134,6 +136,14 @@ def wsgi_app(
             return e(environ, start_response)
         return response(environ, start_response)
 
+    def setup_sentry(self):
```

Ruff (0.8.2)

Missing return type annotation for public function `setup_sentry`

Add return type annotation: `None`

(ANN201)